editor/editor_file_system.cpp

```diff
@@ -2006,6 +2004,31 @@ void EditorFileSystem::_update_script_documentation() {
 	update_script_paths_documentation.clear();
 }
 
+bool EditorFileSystem::_should_reload_script(const String &p_path) {
+	if (first_scan) {
+		return false;
+	}
+
+	Ref<Resource> res = ResourceCache::get_ref(p_path);
+	if (res.is_null()) {
+		// Script not already loaded.
+		return false;
+	}
+
+	Ref<Script> scr = res;
+	if (scr.is_null()) {
+		// Not a script.
+		return false;
+	}
```

Comment:
You can merge these 2 checks.
```suggestion
	Ref<Script> scr = ResourceCache::get_ref(p_path);
	if (scr.is_null()) {
		// Not a script.
		return false;
	}
```

Reply:
Done!